spec/unit/installer/project_cache/project_cache_analyzer_spec.rb

```diff
@@ -118,6 +126,26 @@ module ProjectCache
             result.pod_targets_to_generate.should.equal([])
             result.aggregate_targets_to_generate.should.equal([])
           end
+
+          it 'returns a pod if its target support dir is dirty' do
+            FileUtils.rm_rf @orange_lib.support_files_dir
+            cache_key_target_labels = Hash[@pod_targets.map { |pod_target| [pod_target.label, TargetCacheKey.from_pod_target(pod_target)] }]
+            cache = ProjectInstallationCache.new(cache_key_target_labels, @build_configurations, @project_object_version)
+            analyzer = ProjectCacheAnalyzer.new(@sandbox, cache, @build_configurations, @project_object_version, @pod_targets, [])
+            result = analyzer.analyze
+            result.pod_targets_to_generate.should.equal([@orange_lib])
+            result.aggregate_targets_to_generate.should.equal(nil)
```

Comment:
`.should.be nil`

Reply:
still not fixed?

Reply:
Ah i fixed the one below and not this one